docs/content/docs/advanced/operation-api.md

```diff
@@ -70,7 +70,7 @@ cs.find(task.mixin.VersionedTask, { version: { version: "1.0.0" } })
 
 ```typescript
 export interface OperationProtocol {
-	create<T extends Doc> (_class: Ref<Class<T>>, values: AnyLayout | Doc): Promise<T>
+	create<T extends Doc> (_class: Ref<Class<T>>, values: Partial<Doc>): Promise<T>
```

Comment:
Why `Partial<Doc>` and not `Partial<T>`? :) Also why `Partial<T>` and not `T`? :)

Reply:
Good catch, will fix

Reply:
That was a question actually :) `Partial<T>` or `Omit<T, keyof Doc>`?

Reply:
Omit<T, keyof Doc> is good, but we need a _space to be specified. So I suppose we should add _space as parameter for create,update,etc and have Omit<T, keyof Doc>.

Reply:
Introduced DocumentValue<T> to support embedded fields without _class specified.